cli/kubectl_kadalu/storage_remove.py

```diff
@@ -0,0 +1,192 @@
+"""
+'storage-remove' sub command
+"""
+
+# noqa # pylint: disable=duplicate-code
+# noqa # pylint: disable=too-many-branches
+
+#To prevent Py2 to interpreting print(val) as a tuple.
+from __future__ import print_function
+
+import os
+import tempfile
+import sys
+import json
+
+import utils
+from storage_yaml import to_storage_yaml
+
+
+def set_args(name, subparsers):
+    """ add arguments, and their options """
+
+    parser = subparsers.add_parser(name)
+    arg = parser.add_argument
+
+    arg(
+        "name",
+        help="Storage Name"
+    )
+    utils.add_global_flags(parser)
+
+
+def validate(args):
+    """
+    Validate the storage requested to be deleted
+    is present in kadalu configmap or not.
+    Exit if not present.
+    """
+
+    storage_info_data = get_configmap_data(args.name)
+
+    if storage_info_data is None:
+        print("Aborting.....")
+        print("Invalid name. No such storage '%s' in Kadalu configmap." % args.name)
+        sys.exit(1)
+
+
+def get_configmap_data(volname):
+    """
+    Get storage info data from kadalu configmap
+    """
+
+    cmd = ["sudo", "kubectl", "get", "configmap", "kadalu-info", "-nkadalu", "-ojson"]
```

Comment:
_sudo_ not required right?

Reply:
not required. Also use `args.kubectl_cmd` instead of hard coding the command

Reply:
`utils.kubectl_cmd(args) + ["get", "configmap", "kadalu-info", "-nkadalu", "-ojson"]`

Reply:
done